src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/DocumentServiceTest.kt

```diff
@@ -163,8 +163,9 @@ internal class DocumentServiceTest {
     every { s3Service.uploadFile(capture(uploadedToS3DocumentIdSlot), documentBytes) } just runs
     every { documentRepository.save(capture(savedDocumentSlot)) } returns document
 
+    val details = "Some details"
     val actualDocumentId =
-      underTest.storeDocument(recallId, createdByUserId, documentBytes, documentCategory, newFileName)
+      underTest.storeDocument(recallId, createdByUserId, documentBytes, documentCategory, newFileName, details)
```

Comment:
Not this change but less than obvious that `documentCategory` is a reference to a versioned category:
`private val documentCategory = PART_A_RECALL_REPORT`
Maybe `val documentCategory` local to each test is better than how it is in this class now?